### DISCUSSION TITLE
How does EIDE set up support for connecting to Jlink via IP address and port？

### DISCUSSION THREAD

**Discussion by vscodeenv** [github0null/eide#317] [Ideas]
How EIDE is set up to support Jlink connection via IP address and port; IAR and MDK can support this kind of wireless connection

**Comment by github0null** [comment]
If you want to program flash by remote JLink, use Extra CommandLine

https://em-ide.com/en/docs/modules/flasher/#jlink

![image](https://github.com/github0null/eide/assets/43022536/a8b10d9f-7a5d-416f-9b3b-a5e7e1f10851)

Input `-IP <IP/Tunnel/Nickname>[:Port]` like this

```
-IP 192.168.31.24
-IP 192.168.31.24:40000
```
![image](https://github.com/github0null/eide/assets/43022536/9a85fa3e-0d05-4e04-945c-fe0581443c37)

**Reply by vscodeenv** [reply]
Can I implement debug single-step using an external command line？
